docs/source/developer/virtualdom.create.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
```

Comment:
Interesting, the Typescript docs use the `*` syntax, but it looks like the other is supported as well when you set the `allowSyntheticDefaultImports` setting in TypeScript: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/5128

Reply:
I guess I prefer keeping the more verbose syntax, because that is what is most often documented online and works regardless of that TypeScript setting, so is more portable